engine/src/main/java/com/redhat/cloud/notifications/config/EngineConfig.java

```diff
@@ -44,7 +44,7 @@ public class EngineConfig {
     @Deprecated(forRemoval = true, since = "To be removed when we're done migrating to Unleash in all environments")
     boolean asyncAggregation;
 
-    @ConfigProperty(name = "processor.email.aggregation.use-recipients-resolver-clowdapp.enabled", defaultValue = "false")
+    @ConfigProperty(name = "processor.email.aggregation.use-recipients-resolver-clowdapp.enabled", defaultValue = "true")
```

Comment:
This looks unrelated and could be a breaking change in some environments.

Reply:
That's the idea, but I suppose it have to be validated on dedicated stage env before pomoting it to prod. May by I'm expecting too much, let's rollback it ;)